WordPress/Classes/ViewRelated/Blog/QuickStartChecklistHeader.swift

```diff
@@ -1,6 +1,7 @@
 import Gridicons
 
 class QuickStartChecklistHeader: UIView {
+
```

Comment:
why the space here?

Reply:
This was unintentional - I'll tidy this up.